src/tlo/analysis/utils.py

```diff
@@ -86,6 +86,33 @@ def parse_log_file(log_filepath, level: int = logging.INFO):
     return LogsDict({name: handle.name for name, handle in module_name_to_filehandle.items()}, level)
 
 
+def merge_log_files(log_path_1: Path, log_path_2: Path, output_path: Path) -> None:
+    """Merge two log files, skipping any repeated header lines.
+    
+    :param log_path_1: Path to first log file to merge. Records from this log file will
+        appear first in merged log file.
+    :param log_path_2: Path to second log file to merge. Records from this log file will
+        appear after those in log file at `log_path_1` and any header lines in this file
+        which are also present in log file at `log_path_1` will be skipped.
+    :param output_path: Path to write merged log file to. Can be one of `log_path_1` or
+        `log_path_2` as data is read from files before writing.
+    """
+    with log_path_1.open("r") as log_file_1:
```

Comment:
Have now updated this function to read in log files lines by line. This means it can no longer write out to one of the input files, but as we can always just delete one or both input files after merging this shouldn't be an issue.